src/main/java/cl/transbank/transaccioncompleta/MallFullTransaction.java

```diff
@@ -61,13 +62,13 @@ public static IntegrationType getIntegrationType() {
         }
 
         public static Options buildOptionsForTesting() {
-            return new Options(
+            return new WebpayOptions(
                     "597055555551", "579B532A7440BB0C9079DED94D31EA1615BACEB56610332264630D42D0A36B1C", IntegrationType.TEST);
         }
 
         private static Options buildOptions(Options options) {
             // set default options for Webpay Plus Normal if options are not configured yet
-            if (Options.isEmpty(options) && Options.isEmpty(MallFullTransaction.Transaction.getOptions()))
+            if (WebpayOptions.isEmpty(options) && WebpayOptions.isEmpty(MallFullTransaction.Transaction.getOptions()))
```

Comment:
You could use all `isEmpty` method directly from `Options` based in the previous comment